linode/objcluster/datasource_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build integration
+//go:build (integration || objcluster) && !optional && !long_running
```

Comment:
Same here^

Reply:
Thanks for catching these two.. I must have missed them 👍